### PR TITLE
Under construction

### PR DESCRIPTION
**Pull request by 2bleO**
In the Pull Request we:

Createa and style Home Page
Create and style about page
Pilots section created dynamically

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+* [GitHub](https://www.github.com)
+* [Microverse](https://microverse.org)
```
Comment (vichuge) [inline README.md:46]: - [x] Please, the Acknowledgments section should contain appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the author of the original design, as required by the [Creative Commons License](https://creativecommons.org/licenses/). 🐶 
- I could see you add it on License, and that's pretty good, I invite you to add author credits on acknowledgments and the License on License section.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        <li class="social-icon d-flex"><a href="#"><i class="fab fa-facebook"></i></a></li>
+        <li class="social-icon d-flex"><a href="#"><i class="fab fa-twitter"></i></a></li>
+        <li class="social-icon d-flex"><a href="#">English</a></li>
+        <li class="social-icon d-flex"><a href="#">My Page</a></li>
+        <li class="social-icon d-flex"><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <header class="row justify-content-center">
+      <nav class="navbar flex-md-row navbar-expand-md navbar-light">
+        <a class="navbar-brand d-md-flex flex-md-row-reverse" href="index.html"> SSCGR<br>2021 <i class="fas fa-car"></i></a>
```
Comment (vichuge) [inline index.html:25]: - [x] When I click the logo on the `about` page, I got a 404 error. Please, fix it. 🚗
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+            <ul class="navbar-nav">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="/index.html">Home</a>
```
Comment (vichuge) [inline index.html]: - [x] When I click `Home` on the same home page, I got a 404 error. Please fix it 🐱 
image

File: pages/about.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Super SportCar Global Race</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="../Assets/sass/about.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
+</head>
+<body class="bg-light" id="body">
+  <div class="container">
```
Comment (vichuge) [inline pages/about.html]: - [x] On the `about` page, you're using margin on sides but no on the `Home` page, I invite you to remove the margin on about. 🌵 
- [x] I think for the same reason, the burger icon appears bigger on the `about` page, and the size should be the same for both pages, please fix it. 🦊 
image

File: README.md [hunk 5]
```diff
@@ -1 +1,54 @@
...
+# Global-racing-event
+# Global-racing-event
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Super SportCar Race:
+
+> Web page to Showcase a Fictional Super SportCar Race as Microverse Capstone Project.
+
+![screenshot](/Assets/img/screenshot.png)
```
Comment (vichuge) [inline README.md:10]: - [x] The image you choose on the readme file is bigger than required, I kindly invite you to check that, to have an image of this size could be annoying to the person who checked your project. 🐸
- I add a recommendation (ok well, two recommendations).
image
image

File: Assets/sass/main.scss [hunk 6]
```diff
@@ -0,0 +1,398 @@
...
+  background-color: $dark;
+  padding-bottom: 7.44vh;
+}
+
+.partners-logos {
+  padding: 1.48vh 8.945vw;
+}
+
+/* Media Query */
+@media (min-width: 1px) {
```
Comment (vichuge) [inline Assets/sass/main.scss]: - [x] (Ok, this is a hard one) Please, the CSS styling is built as **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around. In this case, you're using the `@media` for mobile, the default style should be for mobile and the `@media` should be used for desktop, I kindly invite you to fix this. I'm sure you'll find a way to solve it as well as possible. 🦁

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by de-scholar — APPROVED** [review]
Hi @2bleO , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**